src/app/Fake.Runtime/CompileRunner.fs

```diff
@@ -104,12 +106,16 @@ let compile (context:FakeContext) outDll =
     if (destinationFile.Exists) then destinationFile.Delete()
 
     let co = context.Config.CompileOptions
+    let dummyPaketDependencyManagerOption =
+        let currentDir = Path.GetDirectoryName typeof<Marker>.Assembly.Location
+        sprintf "--compilertool:\"%s\"" currentDir
```

Comment:
1. Are the quotes needed?
2. In theory `currentDir` could be `null` if the assembly is ilmerged or something like that, in that scenario we could just leave out the argument and leave it to the caller to provide it (but I'm happy to leave the code as-is, just something I noticed)

Reply:
I was just about to leave a comment asking for your opinion :)

Regarding the quotes, I'm just anticipating spaces in paths. I didn't see quote-handling of arguments in this code path so I was being defensive.

Regarding the assembly location, I could totally check that and omit. If a user is ilmerging then they are going to have more problems because there's no way to specify a dependency resolver other than by file path anyway.

Reply:
Regarding the quotes: I think it should work with spaces just fine given that we work with lists below and provide FSC with a list/array as well? Question is just how FCS API wants these arguments.

Regarding the `null`: I'd leave that up to you to decide.

Reply:
I cleaned this up per your suggestions. Thanks!